examples/lazy_away_step_cg_sparsity.jl

```diff
@@ -1,11 +1,11 @@
 #=
 
-Example demonstrating sparsity control by means of the `lazy_tolerance`-factor passed to the lazy AFW variant
+Example demonstrating sparsity control by means of the `sparsity_control`-factor passed to the lazy AFW variant
```

Comment:
Is this correct here? If this is for a lazified algorithm specifically isn't it still a lazy factor?